eWonConnector-gateway/src/main/java/com/hms_networks/SyncManager.java

```diff
@@ -901,6 +907,30 @@ public QualityCode write(TagPath tagPath, Object o) {
                             provider.updateValue(p, v.getValue(), v.getQuality(), v.getTimestamp());
                         }
 
+                        // Sync tag description to documentation and tooltip
+                        try {
+                            // Get tag provider from gateway context
+                            TagProvider gwTagProvider = gatewayContext.getTagManager().getTagProvider(providerName);
+                            int gwTagProvider_asyncTimeoutSecs = 5;
```

Comment:
This could be final

Reply:
This is also mixing camel case and snake case

Reply:
changed